.devops/code-review-pipelines.yml

```diff
@@ -0,0 +1,56 @@
+trigger:
+  - master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+#variables:
```

Comment:
Why commit commented lines? If there is no need for those instructions simply delete them